Resolve syncing conflicts from repo_sync_working_branch to public

openpublishbuild opened the pull request:
Pull request opened by Docs to resolve syncing conflicts from repo_sync_working_branch to public.

JoeDavies-MSFT committed:
Merged overview

kaarins committed:
Merge branch 'master' into kaarins-patch-3

JoeDavies-MSFT committed:
Overview merge

JoeDavies-MSFT committed:
Add CAE article

JoeDavies-MSFT committed:
Update microsoft-365-continuous-access-evaluation.md

JoeDavies-MSFT committed:
Update TOC.yml

JoeDavies-MSFT committed:
Update TOC.yml

JoeDavies-MSFT committed:
Create m365-defender-locate-threats.svg

JoeDavies-MSFT committed:
Update microsoft-365-defender-train-employees.md

JoeDavies-MSFT committed:
Training updates

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-m365d-solutionize

Dansimp committed:
Update troubleshoot-security-config-mgt.md

Dansimp committed:
Merge branch 'security-config' into patch-1

schmurky committed:
Merge branch 'master' into macky-linktoincident

schmurky committed:
Add ID

schmurky committed:
Add ID

JoeDavies-MSFT committed:
Update incident-response-overview.md

JoeDavies-MSFT committed:
Update incident-response-overview.md

JoeDavies-MSFT committed:
Update incident-response-overview.md

JoeDavies-MSFT committed:
Update incident-response-overview.md

JoeDavies-MSFT committed:
Update incident-response-overview.md

JoeDavies-MSFT committed:
Remove training and move tshoot

JoeDavies-MSFT committed:
Final changes

mjcaparas committed:
Merge branch 'master' into security-config

mjcaparas committed:
update sec config page

mjcaparas committed:
updates

mjcaparas committed:
add related topic

mjcaparas committed:
add related topic

mjcaparas committed:
add applies to

mjcaparas committed:
edits

schmurky committed:
Merge branch 'master' into macky-linktoincident

dlmsft committed:
Update security-config-management.md

dlmsft committed:
Update troubleshoot-security-config-mgt.md

dlmsft committed:
Update troubleshoot-security-config-mgt.md

mjcaparas committed:
Update troubleshoot-security-config-mgt.md

mjcaparas committed:
Merge pull request #11194 from dlmsft/patch-9

Update troubleshoot-security-config-mgt.md

mjcaparas committed:
Merge branch 'master' into security-config

mjcaparas committed:
fix table

mjcaparas committed:
testing

mjcaparas committed:
Merge pull request #11191 from dlmsft/patch-6

Update security-config-management.md

mjcaparas committed:
Merge branch 'security-config' into patch-8

mjcaparas committed:
Merge pull request #11192 from dlmsft/patch-8

Update troubleshoot-security-config-mgt.md

mjcaparas committed:
fix links

mjcaparas committed:
Merge branch 'security-config' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into security-config

mjcaparas committed:
updates

mjcaparas committed:
udpates

dlmsft committed:
Update security-config-management.md

dlmsft committed:
Update troubleshoot-security-config-mgt.md

dlmsft committed:
Update security-config-management.md

mjcaparas committed:
Merge branch 'master' into security-config

JoeDavies-MSFT committed:
Update incident-response-overview.md

mjcaparas committed:
fix blocking issues

JoeDavies-MSFT committed:
Update incident-response-overview.md

mjcaparas committed:
Merge branch 'security-config' into patch-1

mjcaparas committed:
Merge pull request #11087 from matt-call/patch-1

Correction

mjcaparas committed:
Merge pull request #11215 from dlmsft/patch-12

Update security-config-management.md

mjcaparas committed:
Merge branch 'security-config' into patch-11

mjcaparas committed:
Merge pull request #11213 from dlmsft/patch-11

Update troubleshoot-security-config-mgt.md

mjcaparas committed:
Merge branch 'security-config' into patch-10

mjcaparas committed:
Merge pull request #11212 from dlmsft/patch-10

Update security-config-management.md

mjcaparas committed:
add includes

mjcaparas committed:
Merge branch 'security-config' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into security-config

mjcaparas committed:
add includes prereq

mjcaparas committed:
edits

mjcaparas committed:
add prereqs

mjcaparas committed:
add includes

mjcaparas committed:
edit

schmurky committed:
Merge branch 'master' into macky-linktoincident

schmurky committed:
change picture

schmurky committed:
Update

JoeDavies-MSFT committed:
ransomware metadata tag

JoeDavies-MSFT committed:
Ransomware tag

Dansimp committed:
Update whats-new.md

Dansimp committed:
Update whats-new.md

mjcaparas committed:
testing content

mjcaparas committed:
remove known issues section

mjcaparas committed:
add image

mjcaparas committed:
move

mjcaparas committed:
fix images

mjcaparas committed:
fix images

mjcaparas committed:
update includes

mjcaparas committed:
updates

markjjo committed:
fixed query as per customer feedback

markjjo committed:
Merge branch 'master' into markjjo-working

Commit:
add examples for Yammer retention policy timings

Dansimp committed:
author updates

schmurky committed:
Merge branch 'master' into macky-linktoincident

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-CAE

JoeDavies-MSFT committed:
Update microsoft-365-continuous-access-evaluation.md

JoeDavies-MSFT committed:
Merge branch 'Josephd-CAE' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into Josephd-CAE

JoeDavies-MSFT committed:
Update identity-access-prerequisites.md

v-dihans committed:
Merge branch 'master' into dansimp-metadata5

mjcaparas committed:
Merge branch 'master' into security-config

mjcaparas committed:
update troubleshoot

mjcaparas committed:
update link

v-dihans committed:
Merge branch 'master' into dansimp-metadata5

markjjo committed:
Merge branch 'master' into markjjo-working

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-working-new

v-dihans committed:
Merge branch 'master' into dansimp-metadata5

markjjo committed:
Merge branch 'master' into markjjo-working

LoreenLa committed:
Updated insights links

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-m365d-solutionize

markjjo committed:
Merge branch 'master' into markjjo-working

LoreenLa committed:
Update index.yml

chrisda committed:
Update quarantine-email-messages.md

transport rules = 30 days

dlmsft committed:
Update troubleshoot-security-config-mgt.md

Duplicate - this note already appears a few lines before as an 'Important' item

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into cabailey-m365docs-yammerexamples

LoreenLa committed:
Merge branch 'master' into loreenl-viva

LoreenLa committed:
Merge branch 'master' into loreenl-viva

JoeDavies-MSFT committed:
Final tweaks

JoeDavies-MSFT committed:
Delete overview-security-center.md

v-dihans committed:
Merge branch 'master' into dansimp-metadata5

JoeDavies-MSFT committed:
Update incident-response-overview.md

LoreenLa committed:
Update setup-microsoft-viva.md

LoreenLa committed:
Merge branch 'loreenl-viva' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into loreenl-viva

chrisda committed:
Recreating changes from PR #6743 in the public repo

JoeDavies-MSFT committed:
Update microsoft-365-defender.md

chrisda committed:
Update TOC.yml

Typo fix

mjcaparas committed:
update supported platforms

chrisda committed:
Merge branch 'master' into chrisda

LoreenLa committed:
Merge branch 'master' into loreenl-viva

v-dihans committed:
Merge branch 'master' into dansimp-metadata5

markjjo committed:
Merge branch 'master' into markjjo-working

chrisda committed:
Merge branch 'master' into Quarantine-chrisda

pahuijbr committed:
Update configure-server-endpoints.md

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into cabailey-m365docs-yammerexamples

chrisda committed:
Merge pull request #11333 from MicrosoftDocs/Quarantine-chrisda

Update quarantine-email-messages.md

chrisda committed:
Merge branch 'master' into chrisda

v-dihans committed:
Merge branch 'master' into loreenl-viva

JoeDavies-MSFT committed:
Update microsoft-365-defender.md

v-dihans committed:
Merge branch 'master' into dansimp-metadata5

JoeDavies-MSFT committed:
Update microsoft-365-defender.md

v-dihans committed:
Merge branch 'master' into patch-37

JoeDavies-MSFT committed:
Update microsoft-365-defender.md

SKjerland committed:
Update index.yml

Per Neha Bhaskar, updated "What's new?" and "Join the partner webinar series" cards.

mjcaparas committed:
Merge pull request #11331 from dlmsft/patch-13

Update troubleshoot-security-config-mgt.md

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-m365d-solutionize

SKjerland committed:
Update index.yml

Fixed em-dashes

mjcaparas committed:
Update security-config-management.md

mjcaparas committed:
Update security-config-management.md

chrisda committed:
Merge pull request #11338 from MicrosoftDocs/chrisda

Recreating changes from PR #6743 in the public repo

SKjerland committed:
Update index.yml

chrisda committed:
Merge pull request #11342 from MicrosoftDocs/master

Master to Live

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into cabailey-m365docs-yammerexamples

SKjerland committed:
Update index.yml

Dansimp committed:
Merge branch 'master' into Dansimp-appg

Dansimp committed:
Update whats-new.md

markjjo committed:
Github issue fixes

markjjo committed:
Merge branch 'master' into markjjo-working

v-dihans committed:
Merge branch 'master' into patch-37

v-dihans committed:
Merge branch 'master' into dansimp-metadata5

v-dihans committed:
Merge branch 'master' into loreenl-viva

SKjerland committed:
Merge branch 'master' into sharik-working

JoeDavies-MSFT committed:
Fixes

SKjerland committed:
Update index.yml

SKjerland committed:
Merge branch 'sharik-working' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into sharik-working

JoeDavies-MSFT committed:
Additional fixes

chrisda committed:
TOC updates

JoeDavies-MSFT committed:
Update .openpublishing.redirection.json

JoeDavies-MSFT committed:
Merge branch 'Josephd-m365d-solutionize' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into Josephd-m365d-solutionize

mjcaparas committed:
Merge branch 'master' into security-config

mjcaparas committed:
fixes

mjcaparas committed:
Merge branch 'security-config' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into security-config

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-m365d-solutionize

chrisda committed:
Merge pull request #11344 from MicrosoftDocs/chrisda

Scheduler TOC updates

SKjerland committed:
Merge branch 'master' into sharik-working

mjcaparas committed:
Merge branch 'master' into security-config

v-dihans committed:
Merge branch 'master' into cabailey-m365docs-yammerexamples

markjjo committed:
Merge branch 'master' into markjjo-working

chrisda committed:
Update index.yml

Added new recurring meetings topic.

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-m365d-solutionize

chrisda committed:
Merge branch 'master' into chrisda

JoeDavies-MSFT committed:
AppG

mjcaparas committed:
Merge pull request #11345 from MicrosoftDocs/security-config

Security config

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-m365d-solutionize

markjjo committed:
Merge branch 'master' into markjjo-working

Dansimp committed:
Merge branch 'master' into Dansimp-appg

markjjo committed:
Merge pull request #11289 from MicrosoftDocs/markjjo-working

Fix to search query based on customer feedback

v-tophillips committed:
ga changes for app governance

v-tophillips committed:
Merge branch 'master' into v-tophillips-appg-redirects

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-working-new

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-working-new

chrisda committed:
Merge branch 'master' into chrisda

kaarins committed:
Merge branch 'master' into kaarins-patch-1

kaarins committed:
Merge branch 'master' into kaarins-patch-3

JoeDavies-MSFT committed:
Update .openpublishing.redirection.json

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-m365d-solutionize

JoeDavies-MSFT committed:
Delete get-incident-notifications.md

Dansimp committed:
Merge branch 'master' into Dansimp-appg

garycentric committed:
Merge branch 'master' into cabailey-m365docs-yammerexamples

garycentric committed:
Merge pull request #11304 from cabailey/cabailey-m365docs-yammerexamples

add examples for Yammer retention policy timings

garycentric committed:
Merge branch 'master' into sharik-working

kaarins committed:
Merge branch 'master' into kaarins-patch-3

garycentric committed:
Merge pull request #11343 from MicrosoftDocs/sharik-working

Updated M365BP landing page

kaarins committed:
Merge branch 'master' into kaarins-patch-3

garycentric committed:
Merge branch 'master' into Josephd-working-new

Dansimp committed:
Merge branch 'master' into Dansimp-appg

v-tophillips committed:
Update compliance-easy-trials.md

Dansimp committed:
Merge pull request #11254 from MicrosoftDocs/Dansimp-appg

Update whats-new.md App G

v-tophillips committed:
Merge branch 'master' into v-tophillips-trials-update

garycentric committed:
Merge branch 'master' into Josephd-working-new

garycentric committed:
Merge pull request #11350 from MicrosoftDocs/Josephd-working-new

AppG references in IR content

v-tophillips committed:
Merge branch 'master' into v-tophillips-trials-update

v-tophillips committed:
Merge branch 'master' into v-tophillips-appg-redirects

garycentric committed:
Merge branch 'master' into loreenl-viva

v-tophillips committed:
Merge pull request #11349 from MicrosoftDocs/v-tophillips-appg-redirects

retire and redirect app governance

v-tophillips committed:
Merge branch 'master' into v-tophillips-trials-update

v-tophillips committed:
Merge pull request #11351 from MicrosoftDocs/v-tophillips-trials-update

Update compliance-easy-trials.md

garycentric committed:
Merge branch 'master' into loreenl-viva

garycentric committed:
Merge pull request #11337 from MicrosoftDocs/loreenl-viva

Loreenl viva

garycentric committed:
Merge branch 'master' into patch-37

chrisda committed:
Merge branch 'master' into chrisda

chrisda committed:
Merge pull request #11346 from MicrosoftDocs/chrisda

Chrisda to Master

garycentric committed:
Merge branch 'master' into patch-37

garycentric committed:
Fix indentation of content in list items

garycentric committed:
Adjust indentation of image

chrisda committed:
Update preset-security-policies.md

Added Preview note to BIP

JoeDavies-MSFT committed:
Merge branch 'master' into Josephd-CAE

chrisda committed:
Merge pull request #11353 from MicrosoftDocs/BIP-chrisda

Update preset-security-policies.md

garycentric committed:
Merge branch 'master' into patch-37

garycentric committed:
Merge branch 'master' into kaarins-patch-1

garycentric committed:
Merge pull request #11340 from pahuijbr/patch-37

Update configure-server-endpoints.md

garycentric committed:
Merge branch 'master' into kaarins-patch-1

garycentric committed:
Correct indentation of new note

garycentric committed:
Merge branch 'kaarins-patch-1' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into kaarins-patch-1

garycentric committed:
Merge branch 'master' into dansimp-metadata5

garycentric committed:
Merge pull request #11097 from MicrosoftDocs/kaarins-patch-1

Update m365-campaigns-sign-up.md

garycentric committed:
Merge branch 'master' into dansimp-metadata5

garycentric committed:
Merge pull request #11316 from MicrosoftDocs/dansimp-metadata5

metadata author updates

garycentric committed:
Merge branch 'master' into kaarins-patch-3

garycentric committed:
Merge pull request #11099 from MicrosoftDocs/kaarins-patch-3

Update what-is-a-domain.md

Commit:
Merge branch 'master' into Josephd-CAE

Commit:
Merge pull request #11321 from MicrosoftDocs/Josephd-CAE

New CAE article for Ignite: Publish 2:00 PM Tue, Nov 2

Commit:
Merge pull request #11352 from MicrosoftDocs/master

M to L

garycentric committed:
Merge branch 'master' into Josephd-m365d-solutionize

officedocspr commented:
## ![STOP](https://docs.microsoft.com/en-us/office/media/internal/prfilecountcheck-stop-sign.png)  Caution: Large pull request

This **public** repo applies limits on the number of files that can be merged in a single pull request (PR):

- **Warning** Warning message and ability to unblock PR using the *Merge large pull request*<sup>\*</sup> label - 30
- **Block** Block message and PR is blocked - 100

This pull request will change **92** files, which exceeds the **warning** limit of **30**.

Before this pull request can be merged, you must review the list of files it contains and their changes to confirm they are correct and intentional. **This includes files you did not change and/or do not own**. If there are changes you aren't familiar with, contact the individuals who submitted the commits included in this pull request and review the changes with them.

**The individual who merges this pull request is responsible for ensuring the changes included in it are correct, and for resolving any issues that might result in merging it.**

### To merge this pull request

If you have determined that the changes in this pull request are correct, add the **Merge large pull request**<sup>\*</sup> label to it, wait for the **max/pr-file-count** check to pass, and then click **Merge**. To add a label, click the gear icon next to **Labels** and then select **Merge large pull request**<sup>\*</sup>.

![Label UI screenshot](https://docs.microsoft.com/en-us/office/media/internal/prfilecountcheck-label-screenshot.gif)

### To reject this pull request

If you can't confirm the changes in this pull request are correct, click **Close pull request** at the bottom of this page to close it without merging. Work with your team to revert any changes that aren't correct.

<sup>\*</sup> You must have write or triage access to this repo to add labels.

garycentric committed:
Merge pull request #11341 from MicrosoftDocs/Josephd-m365d-solutionize

M365D TOC and content updates for solutions

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

denisebmsft committed:
Merge pull request #11354 from MicrosoftDocs/mdav-relnotes

Update manage-updates-baselines-microsoft-defender-antivirus.md

garycentric committed:
Merge pull request #11355 from MicrosoftDocs/master

Publish 11/02/2021, 3:30 PM

officedocspr commented:
## ![STOP](https://docs.microsoft.com/en-us/office/media/internal/prfilecountcheck-stop-sign.png)  Too many files - PR blocked

This **public** repo applies limits on the number of files that can be merged in a single pull request (PR):

- **Warning** Warning message and ability to unblock PR using the *Merge large pull request*<sup>\*</sup> label - 30
- **Block** Block message and PR is blocked - 100

This pull request will change **116** files, which exceeds the **block** limit of **100**. **This PR will not be merged**.

### To merge this pull request

You must either reduce the number of files being changed to below the 100 file limit or split the changes into multiple PRs. PRs must not have more than 100 files and fewer than 30 files if possible.

### To reject this pull request

If you want to abandon these changes and not merge this PR, click **Close pull request** at the bottom of this page to close it without merging. Work with your team to revert any changes that aren't correct.

<sup>\*</sup> You must have write or triage access to this repo to add labels.

garycentric committed:
Merge branch 'master' into repo_sync_working_branch

garycentric committed:
Merge pull request #11327 from MicrosoftDocs/repo_sync_working_branch

Confirm merge from repo_sync_working_branch to master to sync with https://github.com/MicrosoftDocs/microsoft-365-docs (branch public)

garycentric committed:
Delete duplicated file with misspelling in its name

This file was added in commit https://github.com/MicrosoftDocs/microsoft-365-docs/pull/6743/commits/9812e9bff81f50448c299da207fd9b450e934db6 and replaced by commit https://github.com/MicrosoftDocs/microsoft-365-docs-pr/pull/11338/commits/396dd77b37f92c0db4e36ae92a8ba5e46c84555a

garycentric committed:
Merge pull request #11356 from MicrosoftDocs/v-gmoor-delete-scheduler-reccuring-meetings-md

Delete duplicated file with misspelling in its name

schmurky committed:
Merge branch 'master' into macky-linktoincident

Dansimp committed:
Merge pull request #11357 from MicrosoftDocs/macky-linktoincident

link to incident feature

Dansimp committed:
Merge pull request #11359 from MicrosoftDocs/master

M to l

officedocspr commented:
## ![STOP](https://docs.microsoft.com/en-us/office/media/internal/prfilecountcheck-stop-sign.png)  Too many files - PR blocked

This **public** repo applies limits on the number of files that can be merged in a single pull request (PR):

- **Warning** Warning message and ability to unblock PR using the *Merge large pull request*<sup>\*</sup> label - 30
- **Block** Block message and PR is blocked - 100

This pull request will change **139** files, which exceeds the **block** limit of **100**. **This PR will not be merged**.

### To merge this pull request

You must either reduce the number of files being changed to below the 100 file limit or split the changes into multiple PRs. PRs must not have more than 100 files and fewer than 30 files if possible.

### To reject this pull request

If you want to abandon these changes and not merge this PR, click **Close pull request** at the bottom of this page to close it without merging. Work with your team to revert any changes that aren't correct.

<sup>\*</sup> You must have write or triage access to this repo to add labels.
